ci/deploy-vmck

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -ex
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+export VAGRANT_DOTFILE_PATH=$(mktemp -d --tmpdir VAGRANT_DOTFILE_XXXXXXXX)
```

Comment:
What does this line do?

Reply:
In theory this line tells vagrant where to put it's `.vagrant` folder but in practice it fails. The arguments are not right or something.